python/test/interpolation/Test_scion_interpolation_histogram.py

```diff
@@ -0,0 +1,25 @@
+# standard imports
+import unittest
+
+# third party imports
+
+# local imports
+from scion.interpolation import histogram
+
+class Test_scion_interpolation_histogram( unittest.TestCase ) :
+    """Unit test for the horner function."""
```

Comment:
Is this a unit test for the Horner function or for soemthing else?

Reply:
That's a typo, probably a copy/paste error on my part. I fixed that.